app/backend/src/couchers/models.py

```diff
@@ -263,6 +275,10 @@ class User(Base):
         ),
     )
 
+    @property
+    def timezone(self):
+        return self.timezone_area.tzid if self.timezone_area else "Etc/UTC"
```

Comment:
Don't the timezone areas cover the whole earth's surface?

Reply:
They ought to. But the main thing is that some users still (!!) don't have a `geom`/coordinate. So this mostly covers that case.